add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -29,14 +29,16 @@ const TWO_WAY_MEDIA_CONN_LENGTH = {
  * Values that we segment sharing a room URL action telemetry probes into.
  *
  * @type {{COPY_FROM_PANEL: Number, COPY_FROM_CONVERSATION: Number,
- *   EMAIL_FROM_CALLFAILED: Number, EMAIL_FROM_CONVERSATION: Number}}
+ *    EMAIL_FROM_CALLFAILED: Number, EMAIL_FROM_CONVERSATION: Number,
+ *    FACEBOOK_FROM_CONVERSATION: Number, EMAIL_FROM_PANEL: Number}}
```

Comment:
Just make this `@type` a `@enum {Number}` to avoid the duplicated constant names.